TidepoolOnboardingKitUI/Views/OnboardingSectionPageView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  OnboardingSectionPageView.swift
+//  TidepoolOnboardingKitUI
+//
+//  Created by Darin Krauss on 3/15/21.
+//  Copyright © 2021 Tidepool Project. All rights reserved.
+//
+
+import SwiftUI
+
+struct OnboardingSectionPageView<Destination: View, Content: View>: View {
+    @EnvironmentObject var onboardingViewModel: OnboardingViewModel
+
+    @Environment(\.presentationMode) var presentationMode
+    @Environment(\.complete) var complete
+    @Environment(\.dismiss) var dismiss
+
+    @State private var isCloseAlertPresented = false
+
+    private let section: OnboardingSection
+    private let backButtonHidden: Bool
+    private let destination: Destination?
+    private let content: Content
+
+    init(section: OnboardingSection, backButtonHidden: Bool = false, destination: Destination, @ViewBuilder content: () -> Content) {
+        self.section = section
+        self.backButtonHidden = backButtonHidden
```

Comment:
[nit] I'm guessing there's a reason why you're overriding the default nav bar back button here, but it's not clear to me.  Is it just so you can call `dismiss`? Could you please add code comments somewhere briefly explaining what the code is accomplishing?  Thanks.

Reply:
I'd definitely prefer **not** to have a custom back button here, but I don't have much choice. 

The default behavior of the nav bar (if you don't specify your own back button) is, if the previous view controller's nav bar title is short enough, then use the previous view controller's nav bar title as the back button text, or otherwise just use "Back".

Normally that wouldn't be a bad thing, except that Design explicitly wants the nav bar title for all views within a section to use the section title (and not the title on the view). For example, for all views in the Introduction section the nav bar title will be "Introduction". Using "Introduction" many, many times over for the back button title would just be odd anyhow.

And, yes, there are some of the section titles that are short enough to trigger the default behavior. For example:

<img width="410" alt="Get Looping" src="https://user-images.githubusercontent.com/1224433/112074545-c8007c00-8b33-11eb-8459-6cbb46e8e788.png">

Oh, and as far as the `backButtonHidden` argument here, we want it hidden on the first view of the modal. Back doesn't make sense when you're on the first page.